### ISSUE TITLE
Add “title” attribute to the button block

### ISSUE THREAD

**Issue by yglik** [WordPress/gutenberg#59922]
Hi,
I would like the ability to add a “title” attribute to the button block. 
so the html output will be:
`<a href=“/nowhere” title=“awesome”>button text!</a>`

as also described here
https://wordpress.org/support/topic/add-title-attribute-to-the-button-block/#post-17504173

The title attribute is a native tooltip, it’s good for accessibility and it’s a common attribute that I think should have a place in the core button block.

the implementation shouldn’t be difficult 
It’s adding a block attribute and a hi that can be placed in the “extra” button in the block toolbar
 
![IMG_0309](https://github.com/WordPress/gutenberg/assets/54001035/6dd16249-29ec-4ca8-aa26-68f162f27072)

I would really appreciate insights and thoughts about this subject 
Thanks

**Comment by sahiladit** [comment]
Can you assign it to me?This is my first contribution can you reply me where can i get the code,so that i can edit it.

**Comment by t-hamano** [comment]
Thanks for the report.

Previously, the button block had a UI for setting the title attribute, but it seems to have been removed in #19735 for accessibility reasons.

However, the `title` attribute itself [remains internally](https://github.com/WordPress/gutenberg/blob/998bd28dc839c8cce7dfead44d8bc276661a7e81/packages/block-library/src/button/block.json#L31-L37) for backward compatibility. Therefore, I think that developers who want to provide a UI for the title attribute can do so by hooking the Edit component.

**Comment by sahiladit** [comment]
So should i edit it?

**Comment by yglik** [comment]
> Previously, the button block had a UI for setting the title attribute, but it seems to have been removed in #19735 for accessibility reasons.

What a mind blowing reading that was. 
thank you @t-hamano for that really important information. 

Now instead of spending time building this feature into a plugin I’m going to spend time removing all my title attributes and replacing them in something more accessible. 
just need to figure out what that is 😅

I think this issue can be closed
